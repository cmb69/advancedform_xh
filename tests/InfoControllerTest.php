<?php

namespace Advancedform;

use ApprovalTests\Approvals;
use PHPUnit\Framework\TestCase;

class InfoControllerTest extends TestCase
{
    public function testRendersSystemCheck(): void
    {
        $formGateway = new FormGateway();
        $sut = new InfoController(
            $formGateway,
            "./",
            XH_includeVar("./config/config.php", "plugin_cf")["advancedform"],
            XH_includeVar("./languages/en.php", "plugin_tx")["advancedform"],
        );
        Approvals::verifyHtml($sut->infoAction());
    }
}
