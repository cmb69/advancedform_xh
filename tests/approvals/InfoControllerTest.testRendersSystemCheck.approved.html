<h1>Advancedform 2.4-dev</h1>
<h2>System check</h2><p class="xh_success">PHP version ≥ 7.1.0</p><p class="xh_success">Extension 'ctype' loaded</p><p class="xh_success">Extension 'filter' loaded</p><p class="xh_success">Extension 'hash' loaded</p><p class="xh_success">CMSimple_XH version ≥ 1.7.0</p><p class="xh_fail">Plugin 'Jquery' installed</p><p class="xh_warning">CAPTCHA installed</p><p class="xh_warning">CAPTCHA key configured</p><p class="xh_warning">Folder './advancedform/config/' writable</p><p class="xh_warning">Folder './advancedform/css/' writable</p><p class="xh_warning">Folder './advancedform/languages/' writable</p><p class="xh_success">Folder 'advancedform/data/' writable</p>